watertap/examples/flowsheets/mvc/mvc_single_stage.py

```diff
@@ -688,9 +688,7 @@ def display_metrics(m):
         "Specific energy consumption:              %.2f kWh/m3"
         % value(m.fs.costing.specific_energy_consumption)
     )
-    print(
-        "Levelized cost of water:                  %.2f $/m3" % m.fs.costing.LCOW.value
-    )
+    print("Levelized cost of water:                  %.2f $/m3" % m.fs.costing.LCOW())
```

Comment:
```suggestion
    print("Levelized cost of water:                  %.2f $/m3" % value(m.fs.costing.LCOW)
```
We generally wrap the expression within value() (from pyomo.environ), since this way works with Vars and Expressions.